pkg/extensions/search/schema.graphql

```diff
@@ -19,54 +19,11 @@ type PackageInfo {
      FixedVersion: String 
 }
 
-type ImgResultForCVE {
```

Comment:
Let's do this cleanup as part of https://github.com/project-zot/zot/pull/457.

In this PR, implement just https://github.com/project-zot/zot/pull/696/files#diff-84af1556304ab2f0edbb3541ee90f5e8eb88e4dde65c0a8069e517dea758c210R145, that's the new name of the query you implemented.

Clenan schema.graphql, add that query, and regenerate the code using gqlgen